src/include/network/routines.rb

```diff
@@ -742,24 +756,62 @@ def SetIfaceDown(dev_name)
 
     # Tries to set all interfaces up
     #
-    # @return [boolean] false if some of interfaces cannot be set up
+    # @return [boolean] false if interfaces are empty or some
+    # interface cannot be bring up
```

Comment:
The API is in fact (almost) created from scratch bcs it's original usage was replaced by ```SetIface*``` methods